frontends/concrete-python/Makefile

```diff
@@ -145,6 +145,9 @@ test-notebooks:
 	eval $(shell make silent_cp_activate)
 	./scripts/jupyter/jupyter.sh
 
+test-notebooks-macos:
```

Comment:
Why a specific target for macos?

Reply:
I removed the `eval $(shell make silent_cp_activate)`, as it's done on `pytest-macos` eg. I am not sure it's strictly needed but as I had issues, I thought it could be related. Up to you to try without that, but at least, in my settings, it works.